tensorflow/python/ops/ctc_ops.py

```diff
@@ -308,6 +334,10 @@ def ctc_greedy_decoder(inputs, sequence_length, merge_repeated=True):
     sequence_length: 1-D `int32` vector containing sequence lengths, having size
       `[batch_size]`.
     merge_repeated: Boolean.  Default: True.
+    blank_index: (Optional). Default: num_classes - 1. Define the class index to
```

Comment:
`num_classes - 1`

Reply:
Could you please be more specific? Thanks!

Reply:
use backticks around `num_classes - 1`.

Reply:
Thanks a lot!!